src/atomgraph.jl

```diff
@@ -106,6 +126,7 @@ end
 
 # alternate version where it builds the features too, you have to pass in the results of the make_feature_vectors function
 function add_features!(g::AtomGraph, atom_feature_vecs::Dict{String, Vector{Float32}}, featurization::Vector{AtomFeat})
+    @assert Set(String.(g.elements)) <= Set(keys(atom_feature_vecs)) "Some atoms in your graph do not have corresponding feature vectors! This could be because some features you requested had missing values for these atoms."
```

Comment:
Should we do some exception handling when this happens?

Reply:
I mean that's kind of what the `@assert` is there for, what else are you thinking of? It can't featurize if some features aren't available for some atoms in the graph...

Reply:
I guess it depends on what we want to do if this happens. We can just throw exceptions or fill missing feature values with default values like zeros to continue featurization if those missing values will not affect the featurization result much. There are some similar cases in DeepChem were handled with filling missing values with zeros. So I thought this may be worth for mentioning.